src/index.js

```diff
@@ -0,0 +1,125 @@
+import './style.css';
+import { updateStatus } from './status.js';
+import { load } from './data.js';
+import { makeContainer, makeDrageable } from './dragdrop.js';
```

Comment:
Since you have these functions from different files you could opt for the default import and export.